dds/DCPS/DomainParticipantFactoryImpl.cpp

```diff
@@ -100,7 +100,7 @@ DomainParticipantFactoryImpl::create_participant(
   // needed for the built in topics during enable.
   dp->set_object_reference(dp_obj);  //xxx no change
 
-  if (qos_.entity_factory.autoenable_created_entities == 1) {
+  if (qos_.entity_factory.autoenable_created_entities == true) {
```

Comment:
Your comments here seem to be based on an assumption that I don't think holds.  The code base doesn't currently have one consistent style.  It would be nice (especially if it was a *good* style), but that's not the reality for a project that was developed by a distributed team over 12+ years.

Due to this (and other reasons), there is no requirement that all source code in the project follow one style, however there is a requirement (or at least strong desire) that new changes going in to the repo are as readable as possible and follow the recommended style.

There are always trade-offs here, no solution will be perfect.  But what we've been doing with OpenDDS recently is to make changes incrementally as we're updating sections of the code.  I don't want to have to think about both the "good" style (current) and also keep in mind a bunch of old styles too.  This also avoids huge commits across all source files that can be accidentally destabilizing and also clutter the history with irrelevant changes.